saas/views/metrics.py

```diff
@@ -154,21 +154,13 @@ class RevenueMetricsView(MetricsMixin, TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super(RevenueMetricsView, self).get_context_data(**kwargs)
-        reverse = True
-        account_title = 'Payments'
-        account = Transaction.FUNDS
-        account_table, customer_table, customer_extra = \
-            aggregate_monthly_transactions(self.organization, account,
-                account_title=account_title, from_date=self.ends_at,
-                reverse=reverse)
-        data = SortedDict()
-        data['amount'] = {"title": "Amount",
-                          "unit": "$", "table": account_table}
-        data['customers'] = {"title": "Customers",
-                             "table": customer_table, "extra": customer_extra}
-        context.update({"title": "Revenue Metrics",
-            "data": data,
-            "data_json": json.dumps(data, cls=DjangoJSONEncoder)})
+        context.update({
+            "title": "Revenue Metrics",
+            "tables": [
+                {"title": "Amount", "unit": "$", "key": "amount", "active": True},
```

Comment:
Following comments from [previous pull request](https://github.com/djaodjin/djaodjin-saas/pull/56):
@marmida wrote:
"From where do I get this information? I see that the Charge object has a unit attribute, and that the Organization.withdraw_funds has a hard-coded variable funds_unit that is set to the string usd. Otherwise, I don't see any database fields that would indicate the currency type."

You are correct. We have hard-coded the provider/withdrawal side of the equation ... So far with the Stripe recipients feature, there is no way to transfer amounts into anything else that usd. We will change that once we start to rely on StripeConnect instead of Stripe recipients.

Comment waved.